website/models/course.py

```diff
@@ -0,0 +1,105 @@
+
+    # META CLASS
+    class Meta:
+        verbose_name = 'Videoaula'
+        verbose_name_plural = 'Videoaulas'
+
+    # # TO STRING METHOD
+    # def __str__(self):
+    #     return self.title
+
+
+class Microtopic(models.Model):
+    # DATABASE FIELDS
+    title = models.CharField(verbose_name='Título', max_length=200)
+    description = models.CharField(max_length=500, default="Essa aqui é a sua descrição.", blank=True)
+    video = models.OneToOneField(VideoLesson, verbose_name='Videoaula', on_delete=models.CASCADE)
+    importance = models.SmallIntegerField(verbose_name='Importância')
+    posts = models.ForeignKey(Post, on_delete=models.CASCADE)
+    ratings = models.ForeignKey(Rating, on_delete=models.CASCADE)
+
+    # META CLASS
+    class Meta:
+        verbose_name = 'Microtópico'
+        verbose_name_plural = 'Microtópicos'
+
+    # TO STRING METHOD
+    def __str__(self):
+        return self.title
+
+
+class ExerciseList(models.Model):
+    # DATABASE FIELDS
+    title = models.CharField(verbose_name='Título',
+                             default='Exercícios e exemplos de aplicação.',
+                             max_length=200)
+    url = models.URLField(max_length=200, help_text='URL para o arquivo da Lista de Exercícios.')
+    posts = models.ForeignKey(Post, on_delete=models.CASCADE)
+    ratings = models.ForeignKey(Rating, on_delete=models.CASCADE)
+
+    # META CLASS
+    class Meta:
+        verbose_name = 'Lista de Exercícios'
+        verbose_name_plural = 'Listas de Exercícios'
+
+    # TO STRING METHOD
+    def __str__(self):
+        return self.title
+
+
+class Macrotopic(models.Model):
+    # DATABASE FIELDS
+    title = models.CharField(verbose_name='Título', max_length=200)
+    description = models.CharField(max_length=500, default="Essa aqui é a sua descrição.", blank=True)
+    posts = models.ForeignKey(Post, on_delete=models.CASCADE)
+    ratings = models.ForeignKey(Rating, on_delete=models.CASCADE)
+    microtopic = models.ForeignKey(Microtopic, verbose_name='Microtópico', on_delete=models.CASCADE)
+    exercise_list = models.ForeignKey(ExerciseList,
+                                      verbose_name='Lista de Exercícios',
+                                      on_delete=models.CASCADE,
+                                      null=True,
+                                      blank=True)
+
+    # META CLASS
+    class Meta:
+        verbose_name = 'Macrotópico'
+        verbose_name_plural = 'Macrotópicos'
+
+    # TO STRING METHOD
+    def __str__(self):
+        return self.title
+
+
+class Course(models.Model):
+    # DATABASE FIELDS
+    title = models.CharField(verbose_name='Título', max_length=200)
+    description = models.CharField(max_length=500, default="Essa aqui é a sua descrição.", blank=True)
+    macrotopic = models.ForeignKey(Macrotopic, verbose_name='Macrotópico', on_delete=models.CASCADE)
+    posts = models.ForeignKey(Post, on_delete=models.CASCADE)
```

Comment:
This causes an error when try to create a course on admin page because these fields doesnt have the property ```null=True``` and they are not shown in the page that creates a course, consequently, these fields dont get any value and they are null.

In fact, these values are not defined at the creation of the course, so it should be null at the creation of the object;

```suggestion
    posts = models.ForeignKey(Post, on_delete=models.CASCADE, null=True)
```